### DISCUSSION TITLE
Is there a way to make the websites light by default, and the button just ADDS dark mode to certain ones?

### DISCUSSION THREAD

**Discussion by jjrmm7** [darkreader/darkreader#10196] [Q&A]
At this point everything is dark mode enabled, and then I disable websites that either don't look good in dark mode, or don't need it based on how bright it personally is. Is there a setting so the app is OFF by default and I only enable ones I need it for? Thank you

**Comment by Gusted** [comment]
Please look at https://github.com/darkreader/darkreader/discussions/9016

**Reply by Gusted** [reply]
Or you could enable the "Invert listed only":
![image](https://user-images.githubusercontent.com/25481501/200189886-8c55e8a3-361d-42a2-9433-b686f6b5425c.png)
